lib/system/widestrs.nim

```diff
@@ -17,6 +17,8 @@ type
   Utf16Char* = distinct int16
   WideCString* = ref UncheckedArray[Utf16Char]
 
+proc ord*(arg: Utf16Char): int = int(cast[uint16](arg))
```

Comment:
Meh, don't export it, bugfixes shouldn't add new procs.

Reply:
yes I can unexport it again, would not be a problem. The question is, should it be exported? If you ask me, then yes it should.

Reply:
It shouldn't. ``ord`` is what it is and it's not some user-extensible conversion proc. It's bad enough that it exists at all, it serves little purpose over a type conversion to ``int``.